src/core.jsx

```diff
@@ -43,14 +54,14 @@ export default (
       return !wait
     }
 
-    omitLoadInProps = (props) => {
-      const isLoadAFunction = isFunction(props.load)
+    omitLoadInProps = (props, loadFunctionName) => {
```

Comment:
You add a parameter here, but you can retrieve the loadFunctionName from configuration.

Look at `componentWillReceiveProps`, the hoc is inconsistent : 
 - I know the load function is not called after the first mount
 - I prefer that the hoc still be consistent over its lifetime